augur/utils.py

```diff
@@ -126,6 +126,11 @@ def write_json(data, file_name, indent=(None if os.environ.get("AUGUR_MINIFY_JSO
         json.dump(data, handle, indent=indent, sort_keys=sort_keys, cls=AugurJSONEncoder)
 
 
+def json_size(data, indent=2):
+    """Return size in bytes of a Python object in JSON string form."""
+    return len(json.dumps(data, indent=indent, cls=AugurJSONEncoder).encode("utf-8"))
```

Comment:
As we normally serialize to JSON directly to a file handle, serializing here to a Python string (which we immediately throw away!) will dramatically increase the memory footprint of `augur export`...

Reply:
Good point. This is not released yet - I've opened an issue which should be addressed before release: #1385